test/integration-tests/IntegrationTestsSample/src/RazorPagesProject/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace RazorPagesProject.Data
```

Comment:
If you change all of the namespaces to [File scoped namespaces ](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces)
`namespace RazorPagesProject.Data`
VS will format the code for you.

Reply:
Updated all .cs files to use file scoped namespaces. Also aligned the namespaces with the folder structure which was not consistent across files.